integration-tests/trait-dyn-cross-contract-calls/contracts/incrementer-caller/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::new_without_default)]
+
+#[ink::contract]
+pub mod caller {
+    use dyn_traits::Increment;
+
+    /// The caller of the incrementer smart contract.
+    #[ink(storage)]
+    pub struct Caller {
+        /// Here we generate and accept a type which implements the `Incrementer` ink! trait.
```

Comment:
```suggestion
        /// Here we accept a type which implements the `Incrementer` ink! trait.
```

Looking back at my suggestion this isn't correct, since the macro is only pointing to a type which was already generated by the codegen